src/Symfony/Component/Console/Output/ConsoleSectionOutput.php

```diff
@@ -63,10 +63,8 @@ public function clear(int $lines = null)
 
     /**
      * Overwrites the previous output with a new message.
-     *
-     * @param array|string $message
      */
-    public function overwrite($message)
+    public function overwrite(array|string $message)
```

Comment:
```suggestion
    public function overwrite(iterable|string $message)
```

(consistent with `OutputInterface::writeln()`)